.github/CONTRIBUTING.md

```diff
@@ -13,6 +13,7 @@ If you find a bug in the source code, you can help us by submitting an issue to
 Please include unit tests where necessary to cover any functionality that is introduced.
 
 ## Coding Guidelines
+* * **See our [Android SDK Development guidelines](https://github.com/rakutentech/mobile-sdk-guidelines/blob/master/android-sdk-development-guidelines.md)**
```

Comment:
<img width="448" alt="Screen Shot 2020-11-24 at 15 34 02" src="https://user-images.githubusercontent.com/66930373/100057309-87854000-2e6a-11eb-878c-87fe87b34b77.png">
I think, we can remove the single * to remove this additional space.

Reply:
Thanks, I missed this.